jax_distribution/systems/ippo_ff_anakin.py

```diff
@@ -141,16 +154,16 @@ class Transition(NamedTuple):
 
 class ActorCritic(nn.Module):
     """Actor Critic Network."""
-
     action_dim: Sequence[int]
+    env_name: str
     activation: str = "tanh"
-    env_name: str = "RobotWarehouse-v0"
 
     @nn.compact
     def __call__(self, observation) -> Tuple[distrax.Categorical, jnp.ndarray]:
-        """Forward pass."""
+
         x = process_observation(observation, self.env_name)
 
+        """Forward pass."""
```

Comment:
this comment was added as a general info about the function maybe we need to keep it in the top?

Reply:
Updated :+1: